### DISCUSSION TITLE
Pengumpulan LBB

### DISCUSSION THREAD

**Discussion by mosesnugraha** [teamalgoritma/community#585] [Q&A] [answered]
selamat malam, saya mau tanya, halo LBBnya belum selesai tapi udah disubmit karena waktunya ga jukup apakah jadi penilaian? dan seandainya LBBnya diperbaiki lagi dań disubmit lagi di luar batas waktu yang ditentukan, apanha mais dalam penilaian? karena LBB saya ada yang mau saya tanyakan lagi dikelas

**Comment by idoali** [comment, marked as answer]
Selamat pagi pak 

Terima kasih karena sudah mengumpulkan LBB pak. 

Nantinya LBB akan kami periksa dan kami berikan feedback pak. Kemudian bapak bisa mengirimkan lagi LBB yang sudah direvisi berdasarkan feedback dari kami, jika bapak mau. LBB yang kami nilai sendiri itu hanya LBB Interactive Plotting pak, dan nilainya bisa berubah setelah revisi dari bapak
